spec/tokenized-buffer-spec.coffee

```diff
@@ -900,3 +900,17 @@ describe "TokenizedBuffer", ->
       expect(tokenizedBuffer.tokenizedLineForRow(1).tokens[0].value).toBe 'b'
       expect(tokenizedBuffer.tokenizedLineForRow(2).tokens.length).toBe 1
       expect(tokenizedBuffer.tokenizedLineForRow(2).tokens[0].value).toBe 'c'
+
+  describe ".parseIndentation(leadingWhitespace)", ->
```

Comment:
`parseIndentation` returns [two parameters](https://github.com/atom/atom/pull/5658/files#diff-dc7bdf9554a6b952bca110e83059dae8R367), `indentLevel` and `excessSpace`. Do you think it would make sense to extend these specs so that both returned parameters are covered (currently, `excessSpace` isn't, as far as I can tell)? That would prevent future :octocat: from breaking the behavior of `parseIndentation` which you're introducing in this PR.